client/packages/programs/src/JsonForms/common/components/Text.tsx

```diff
@@ -147,6 +150,8 @@ const UIComponent = (props: ControlProps) => {
   }
 
   const width = schemaOptions?.width ?? '100%';
+  const multiline = schemaOptions?.multiline !== false;
+  const rows = schemaOptions?.rows;
```

Comment:
`multiline` is ON by default, which I think is okay. There's no difference visually -- it just expands to a new line if the text needs to wrap.

`rows` is ignored if `multiline == false`